packages/shared/components/modals/ExpirationTimePickerMenu.svelte

```diff
@@ -1,5 +1,11 @@
 <script lang="typescript" context="module">
-    export type ExpirationTimeSelection = 'none' | '1hour' | '1day' | '1week' | 'custom'
+    export enum ExpirationTime {
+        None,
+        OneHour,
+        OneDay,
+        OneWeek,
+        Custom,
```

Comment:
Do we need the enum variant values too? Doesn't look like they're used for the locales, but just making sure.

Reply:
no we dont need them